tests/ctst/steps/azureArchive.ts

```diff
@@ -148,25 +151,50 @@ async function cleanZenkoBucket(
     world: Zenko,
     bucketName: string,
 ): Promise<void> {
+    if (!bucketName) {
+        return;
+    }
     world.resetCommand();
     world.addCommandParameter({ bucket: bucketName });
-    const results = await S3.listObjectVersions(world.getCommandParameters());
-    const res = safeJsonParse(results.stdout);
-    assert(res.ok);
-    const parsedResults = res.result as listingResult;
-    const versions = parsedResults.Versions || [];
-    const deleteMarkers = parsedResults.DeleteMarkers || [];
-    await Promise.all(versions.concat(deleteMarkers).map(obj => {
-        world.addCommandParameter({ key: obj.Key });
-        world.addCommandParameter({ versionId: obj.VersionId });
-        return S3.deleteObject(world.getCommandParameters());
-    }));
-    world.deleteKeyFromCommand('key');
-    world.deleteKeyFromCommand('versionId');
+    const createdObjects = world.getSaved<Map<string, string>>('createdObjects');
+    if (createdObjects !== undefined) {
+        const results = await S3.listObjectVersions(world.getCommandParameters());
```

Comment:
We need to list objects to delete all objects versions including deleteMarkers, because we add versionId to savedObject when putting the object the first time, but not when adding user MD to it nor do we save the deleteMarker when deleting an object... That said we can probably just list objects instead of checking `world.getSaved` which is not a big deal i think..